src/OpenTK/Platform/SDL2/Sdl2DisplayDeviceDriver.cs

```diff
@@ -61,9 +61,15 @@ public Sdl2DisplayDeviceDriver()
                     current_mode.Width, current_mode.Height,
                     TranslateFormat(current_mode.Format),
                     current_mode.RefreshRate);
-
+                float pixelScaleD, pixelScaleW, pixelScaleH;
+                //Get display DPI from SDL
+                SDL.GetDisplayDPI(d, out pixelScaleD, out pixelScaleW, out pixelScaleH);
+                //Convert to scale factor using default 96DPI
+                pixelScaleD /= 96.0f;
+                pixelScaleW /= 96.0f;
+                pixelScaleH /= 96.0f;
```

Comment:
`pixelScaleD`, `pixelScaleW`, and `pixelScaleH` are never used here, guessing this is a mistake?